ui/app/AppLayouts/Profile/panels/ProfileDescriptionPanel.qml

```diff
@@ -48,15 +48,24 @@ Item {
 
             label: qsTr("Bio")
             placeholderText: qsTr("Tell us about yourself")
+            input.maximumLength : 0
```

Comment:
It feels weird to set this to 0; I'd set it to the max value, which is according to docs and source code:
> By default, this property contains a value of 32767.